[2023-09-04T06:03:15.224+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dag_with_postgres_operator_v04.insert_into_table scheduled__2022-05-30T00:00:00+00:00 [queued]>
[2023-09-04T06:03:15.231+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dag_with_postgres_operator_v04.insert_into_table scheduled__2022-05-30T00:00:00+00:00 [queued]>
[2023-09-04T06:03:15.232+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 6
[2023-09-04T06:03:15.245+0000] {taskinstance.py:1382} INFO - Executing <Task(PostgresOperator): insert_into_table> on 2022-05-30 00:00:00+00:00
[2023-09-04T06:03:15.250+0000] {standard_task_runner.py:57} INFO - Started process 5769 to run task
[2023-09-04T06:03:15.254+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'dag_with_postgres_operator_v04', 'insert_into_table', 'scheduled__2022-05-30T00:00:00+00:00', '--job-id', '631', '--raw', '--subdir', 'DAGS_FOLDER/dag_with_postgres_operator.py', '--cfg-path', '/tmp/tmp24guf9qo']
[2023-09-04T06:03:15.256+0000] {standard_task_runner.py:85} INFO - Job 631: Subtask insert_into_table
[2023-09-04T06:03:15.310+0000] {task_command.py:415} INFO - Running <TaskInstance: dag_with_postgres_operator_v04.insert_into_table scheduled__2022-05-30T00:00:00+00:00 [running]> on host 10bdc2e98230
[2023-09-04T06:03:15.396+0000] {taskinstance.py:1660} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='coder2j' AIRFLOW_CTX_DAG_ID='dag_with_postgres_operator_v04' AIRFLOW_CTX_TASK_ID='insert_into_table' AIRFLOW_CTX_EXECUTION_DATE='2022-05-30T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2022-05-30T00:00:00+00:00'
[2023-09-04T06:03:15.397+0000] {sql.py:274} INFO - Executing: 
            insert into dag_runs (dt, dag_id) values ('2022-05-30', 'dag_with_postgres_operator_v04')
        
[2023-09-04T06:03:15.404+0000] {base.py:73} INFO - Using connection ID 'postgres_localhost' for task execution.
[2023-09-04T06:03:15.410+0000] {base.py:73} INFO - Using connection ID 'postgres_localhost' for task execution.
[2023-09-04T06:03:15.420+0000] {sql.py:418} INFO - Running statement: 
            insert into dag_runs (dt, dag_id) values ('2022-05-30', 'dag_with_postgres_operator_v04')
        , parameters: None
[2023-09-04T06:03:15.423+0000] {sql.py:427} INFO - Rows affected: 1
[2023-09-04T06:03:15.433+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=dag_with_postgres_operator_v04, task_id=insert_into_table, execution_date=20220530T000000, start_date=20230904T060315, end_date=20230904T060315
[2023-09-04T06:03:15.466+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2023-09-04T06:03:15.481+0000] {taskinstance.py:2784} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2023-09-04T06:06:11.411+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dag_with_postgres_operator_v04.insert_into_table scheduled__2022-05-30T00:00:00+00:00 [queued]>
[2023-09-04T06:06:11.442+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dag_with_postgres_operator_v04.insert_into_table scheduled__2022-05-30T00:00:00+00:00 [queued]>
[2023-09-04T06:06:11.443+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 6
[2023-09-04T06:06:11.480+0000] {taskinstance.py:1382} INFO - Executing <Task(PostgresOperator): insert_into_table> on 2022-05-30 00:00:00+00:00
[2023-09-04T06:06:11.494+0000] {standard_task_runner.py:57} INFO - Started process 7112 to run task
[2023-09-04T06:06:11.501+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'dag_with_postgres_operator_v04', 'insert_into_table', 'scheduled__2022-05-30T00:00:00+00:00', '--job-id', '1190', '--raw', '--subdir', 'DAGS_FOLDER/dag_with_postgres_operator.py', '--cfg-path', '/tmp/tmpvvurmmap']
[2023-09-04T06:06:11.507+0000] {standard_task_runner.py:85} INFO - Job 1190: Subtask insert_into_table
[2023-09-04T06:06:11.679+0000] {task_command.py:415} INFO - Running <TaskInstance: dag_with_postgres_operator_v04.insert_into_table scheduled__2022-05-30T00:00:00+00:00 [running]> on host 10bdc2e98230
[2023-09-04T06:06:11.994+0000] {taskinstance.py:1660} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='mohghaff' AIRFLOW_CTX_DAG_ID='dag_with_postgres_operator_v04' AIRFLOW_CTX_TASK_ID='insert_into_table' AIRFLOW_CTX_EXECUTION_DATE='2022-05-30T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2022-05-30T00:00:00+00:00'
[2023-09-04T06:06:11.997+0000] {sql.py:274} INFO - Executing: 
            insert into dag_runs (dt, dag_id) values ('2022-05-30', 'dag_with_postgres_operator_v04')
        
[2023-09-04T06:06:12.025+0000] {base.py:73} INFO - Using connection ID 'postgres_localhost' for task execution.
[2023-09-04T06:06:12.051+0000] {base.py:73} INFO - Using connection ID 'postgres_localhost' for task execution.
[2023-09-04T06:06:12.087+0000] {sql.py:418} INFO - Running statement: 
            insert into dag_runs (dt, dag_id) values ('2022-05-30', 'dag_with_postgres_operator_v04')
        , parameters: None
[2023-09-04T06:06:12.101+0000] {sql.py:427} INFO - Rows affected: 1
[2023-09-04T06:06:12.145+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=dag_with_postgres_operator_v04, task_id=insert_into_table, execution_date=20220530T000000, start_date=20230904T060611, end_date=20230904T060612
[2023-09-04T06:06:12.360+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2023-09-04T06:06:12.404+0000] {taskinstance.py:2784} INFO - 0 downstream tasks scheduled from follow-on schedule check
